{
    "type": "bundle",
    "id": "bundle--a78b33d8-c2ae-4fd4-a8df-6d901106dafb",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--d06c28aa-cc66-4b26-a311-eac97038ddd0",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-06-09T00:04:48.237273Z",
            "modified": "2022-06-09T00:04:48.237273Z",
            "name": "CVE-2022-31313",
            "description": "api-res-py package in PyPI 0.1 is vulnerable to a code execution backdoor in the request package.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-31313"
                }
            ]
        }
    ]
}